packages/playground/src/shared/notifications/Scene.tsx

```diff
@@ -0,0 +1,232 @@
+import studio from '@theatre/studio'
```

Comment:
Suggestion: simplify this playground example so it has nothing in it except the notifications code. Would help me in the future if I'm looking to reference notification usage and don't have to look through the playground example as much to find it.

Reply:
Alright, I'll simplify it a bit

Reply:
👍🙂

Reply:
Done. I kept the background color code because it helps us easily catch UI issues.